aerogear-android-sdk-bom/pom.xml

```diff
@@ -0,0 +1,118 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+  JBoss, Home of Professional Open Source
+  Copyright Red Hat, Inc., and individual contributors
+
+  Licensed under the Apache License, Version 2.0 (the "License");
+  you may not use this file except in compliance with the License.
+  You may obtain a copy of the License at
+
+      http://www.apache.org/licenses/LICENSE-2.0
+
+  Unless required by applicable law or agreed to in writing, software
+  distributed under the License is distributed on an "AS IS" BASIS,
+  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  See the License for the specific language governing permissions and
+  limitations under the License.
+-->
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+
+    <modelVersion>4.0.0</modelVersion>
+    <groupId>org.jboss.aerogear</groupId>
+    <artifactId>aerogear-android-sdk-bom</artifactId>
+    <version>1.1.7-SNAPSHOT</version>
+    <packaging>pom</packaging>
+
+    <name>JBoss AeroGear Android SDK BOM</name>
+    <description>Bill Of Materials for AeroGear Android SDK</description>
+    <url>http://aerogear.org</url>
+
+    <properties>
+        <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
+
+        <!--  dependencies -->
+        <okhttp.version>3.9.1</okhttp.version>
```

Comment:
Should we prefer ranges, e.g. `[3.9.1, 4.0.0)` (>= 3.9.1 && < 4.0.0)?

Reply:
NOPE ! let's use explicit versions